qcp/QMatrices.py

```diff
@@ -0,0 +1,42 @@
+class QMatrices:
+
```

Comment:
In general since we'll be sharing the code among ourselves, some more comments on what/why wouldn't hurt.